redmine_import_time_entry/unit/mapper.py

```diff
@@ -0,0 +1,101 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright (C) 2015 - Present Savoir-faire Linux
+#    (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.tools.translate import _
+from openerp.addons.connector_redmine.backend import redmine
+from openerp.addons.connector_redmine.unit.mapper import RedmineImportMapper
+from openerp.addons.connector.unit.mapper import mapping
+from openerp.addons.connector.exception import MappingError
+
+
+@redmine
+class TimeEntryImportMapper(RedmineImportMapper):
+    _model_name = 'redmine.hr.analytic.timesheet'
+
+    direct = [
+        ('spent_on', 'date'),
+        ('hours', 'unit_amount'),
+        ('entry_id', 'redmine_id'),
+    ]
+
+    @mapping
+    def name(self, record):
+        name = self.backend_record.name
+
+        issue_id = record['issue_id']
+
+        if issue_id:
+            name += _(' Issue #%s - %s') % (issue_id, record['issue_subject'])
```

Comment:
Why the extra space? Translators might not think to keep it.

If `issue_id` is an int, `%d` should be preferred.

It would be clearer for translators if you used a dictionary formatter:

```python
name += ' ' + _('Issue #%(issue_id)d - %(issue_subject)s') % {
    issue_id=issue_id,
    issue_subject=record['issue_subject'],
}
```

Is the `' - %s'` part subject to translation?